app/jobs/regular/bulk_invite.rb

```diff
@@ -105,17 +119,35 @@ def send_invite(invite)
               end
             end
           end
+
+          if user_fields.present?
+            user_fields.each do |user_field, value|
+              user.custom_fields["#{User::USER_FIELD_PREFIX}#{user_field}"] = value
```

Comment:
We do this a lot. I wonder if it would make sense to add a helper? My feeling is code outside of `user` probably needn't be messing with prefixes. For example:

`user.set_user_field(user_field, value)`